### PR TITLE
Allow to use timeout in function kwargs even when initial seconds is None

### PR DESCRIPTION
**Pull request by roidelapluie**
Initial support for timeout in function kwargs was implemented by 187ee34d9e0a571bcc7f9233208462634940f4c0

### CODE REVIEW

File: timeout_decorator/timeout_decorator.py [hunk 1]
```diff
@@ -65,6 +62,10 @@ def new_function(*args, **kwargs):
                 if new_seconds:
                     old = signal.signal(signal.SIGALRM, handler)
                     signal.setitimer(signal.ITIMER_REAL, new_seconds)
+
+                if not seconds:
```
Comment (pnpnpn) [inline timeout_decorator/timeout_decorator.py]: Why did you move this?

**Comment by pnpnpn** [comment]
Can you fix the pep8 failure?

File: timeout_decorator/timeout_decorator.py (hunk 1 above)
Comment (roidelapluie) [inline timeout_decorator/timeout_decorator.py]: otherwhise we return the original function if the seconds parameter on the Timeout is none at the time we create the decorator.

Comment (roidelapluie) [inline timeout_decorator/timeout_decorator.py]: you will see the problem if you move it back and run the new test I wrote

**Comment by roidelapluie** [comment]
@pnpnpn done

**Comment by maltekliemann** [comment]
@pnpnpn Is this a "won't fix"? It would be good to know at least if the current no-op behavior for `seconds=None` is expected behavior or a bug.

**Comment by pnpnpn** [comment]
The build is broken for the PR

**Comment by maltekliemann** [comment]
@pnpnpn If I'm not mistaken, the build failed in 2016 for py32, and that's it. py32 is long past it's end of life now, and you've even removed it from the tox config. I don't see how this failure is a problem.

**Comment by maltekliemann** [comment]
@pnpnpn Just to be clear, I think it would be good to change the behavior of ``timeout`` as outlined in the pull request, but it would be even better to clarify if the current behavior is expected or buggy.

**Comment by pnpnpn** [comment]
Makes sense. I will merge the PR after I fix the build

**Comment by maltekliemann** [comment]
Thanks!

**Comment by pnpnpn** [comment]
Fixed build with https://github.com/pnpnpn/timeout-decorator/pull/65
